packages/config/config/devices/0x0438/dimmer-400w.json

```
@@ -0,0 +1,116 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "Z-Wave Dimmer 400W",
+	"description": "In-Wall Dimmer Module",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xD005"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_on_prev"
+		},
+		"3": {
+			"label": "Send Basic Report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Default Fade Time",
+			"valueSize": 1,
```

Does this have a unit?

---

Default fade time (unit is second)

---

Then it should be added to the definition.
```suggestion
			"valueSize": 1,
			"unit": "seconds",
```